test/format/config_compat.sed

```diff
@@ -0,0 +1,89 @@
+s/^assert.commit_timestamp=/assert_commit_timestamp=/
```

Comment:
To make this script Evergreen compatible (which has prepended timestamp strings in the CONFIG dump), should we remove `^`?

Reply:
@lukech 
> To make this script Evergreen compatible (which has prepended timestamp strings in the CONFIG dump), should we remove ^?

The anchoring`^` is necessary, some configuration strings are subsets of other configuration strings, people will have to strip the timestamp strings before running the sed script, if it becomes necessary to use those CONFIG files with previous releases.